AGENTS.md

```
@@ -139,6 +139,39 @@ When finishing a development branch:
+| Dependency version bump | 1 |
+| Internal controller refactor (no CRD/behavior change) | 2 |
+| Reconciler logic changes | 2 |
+| RBAC/permissions changes | 3 |
+| CRD schema changes (OLSConfig) | 3 |
+| Upgrade/migration path changes | 3 |
+
+### Jira Description Format
+
+Include this section in every ticket description:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)